docs/quickstart.rst

```diff
@@ -1,10 +1,242 @@
 Quickstart
 ==========
 
+Welcome to the *Septentrion* quickstart documentation.
+
+In this tutorial, you will learn how to write migrations in the expected format, how to visualize them and how to run
+them against a PostgreSQL database.
+
+Prepare your database
+---------------------
+
+If you already have a PostgreSQL database around, make sure to note the connection
+parameters. Otherwise, we'll create one together with Docker_:
+
+.. _Docker: https://docs.docker.com/
+
+.. code-block:: console
+
+    $ docker run --detach --rm -p 5432:5432 --name septentrion_db postgres
+
+.. note::
+
+    If you need to stop the docker at some point, use ``docker stop septentrion_db``.
+
+.. note::
+
+    If you stop the Docker or reboot your machine, the Postgres database will be trashed and you will need to re-create
+    one and add some data again.
```

Comment:
If you just stopped the Postgres container you can restart it without loosing any database data. It is when you remove a container that you loose the data that was on the container file system.

Reply:
I believe this is not true if it's launched as `docker run --rm` as mentioned above.

Reply:
True. I read it too fast.